.github/workflows/pythonpackage.yml

```diff
@@ -0,0 +1,31 @@
+name: Python package
```

Comment:
It is just the name of the CI action. What do you have in mind?

Reply:
I don't know. It seems to do a number of things, none of which are packaging. But then it has to be called something and I'm not coming up with anything better. It can always be changed later if a better option comes along.

Reply:
OK. I can understand where you are coming from. The name comes from the Github starter action for building and testing a Python package (which is sort-of what we are doing.)

Reply:
Lets go with it for the moment.